### DISCUSSION TITLE
Tensorboard logs not showing in HF anymore - ORPO

### DISCUSSION THREAD

**Discussion by amritap-ef** [huggingface/autotrain-advanced#751] [Q&A]
Hi,

My training runs do not seem to be able to be rendered by HF's "Training Metrics" inbuilt tensorboard feature on the model repo. 

Base model being finetuned: Meta-Llama-3.1-8B-Instruct
Task: LLM ORPO
Even though "log" is not available in the params UI, the resulting training_params has "log" set to "tensorboard"

Would be great to be able to see the logs on HF.

<img width="1480" alt="Screenshot 2024-09-03 at 13 29 05" src="https://github.com/user-attachments/assets/349066ed-49c8-45b8-ad9c-30160b8c7c69">

**Comment by abhishekkrthakur** [comment]
they update every 10mins

**Reply by amritap-ef** [reply]
To clarify, this is on a completed training job
